dts/arm/olimex_stm32_e407.dts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2017, Erwin Rol <erwin@erwinrol.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32f407.dtsi>
+
+/ {
+	model = "Olimex STM32-E407 board";
+	compatible = "st,olimex_stm32_e407", "st,stm32f407";
```

Comment:
As Olimex is the vendor this should be:

compatible = "olimex,stm32-e407", "st,stm32f407";